contentrepo/settings/base.py

```diff
@@ -266,5 +266,5 @@
 
 # Flag for turning on Standalone Whatsapp Templates, still in development
 ENABLE_STANDALONE_WHATSAPP_TEMPLATES = env.bool(
-    "ENABLE_STANDALONE_WHATSAPP_TEMPLATES", False
+    "ENABLE_STANDALONE_WHATSAPP_TEMPLATES", True
```

Comment:
I think we should either keep this default to False, or if it breaks things now if set to False, then we should remove it